Gemfile.lock

```diff
@@ -647,9 +649,9 @@ GEM
     zlib (3.1.0)
 
 PLATFORMS
+  arm64-darwin-23
   ruby
   x86_64-darwin-20
-  x86_64-linux
```

Comment:
I think we need `x86_64-linux` for the azure deployment environment

Reply:
I'm not sure why this was removed when I did `bundle install` 🤔 I think it would be added back in as part of the docker build as that does its own `bundle install` but I'll add it back in manually